src/backend/metal/src/device.rs

```diff
@@ -1942,25 +1942,31 @@ impl hal::Device<Backend> for Device {
     fn reset_fence(&self, fence: &n::Fence) {
         *fence.0.borrow_mut() = n::FenceInner::Idle { signaled: false };
     }
-    fn wait_for_fence(&self, fence: &n::Fence, mut timeout_ms: u32) -> bool {
-        debug!("wait_for_fence {:?} for {} ms", fence, timeout_ms);
+    fn wait_for_fence(&self, fence: &n::Fence, mut timeout_ns: u64) -> bool {
+        use std::{time, thread};
```

Comment:
why do we need to have this `use` if those types were already used here without one?

Reply:
it's not needed, I removed it